src/rollback_to_stable/rts_btree.c

```diff
@@ -59,9 +59,7 @@ __rts_btree_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *firs
          */
         txn_id_visible = __wti_rts_visibility_txn_visible_id(session, upd->txnid);
         if (!txn_id_visible || rollback_timestamp < upd->upd_durable_ts ||
-          (upd->prepare_state == WT_PREPARE_INPROGRESS &&
-            (!F_ISSET(conn, WT_CONN_PRESERVE_PREPARED) ||
-              rollback_timestamp < upd->upd_start_ts))) {
```

Comment:
We need to rollback only the updates that are having prepare timestamp > stable timestamp. Removing this condition may rollback the updates that are stable?

Reply:
No, we need to rollback all the prepared in ASC so we don't need the check here if we no longer needs to worry about fuzzy checkpoint.